pages/common/topgrade.md

```diff
@@ -0,0 +1,28 @@
+# Topgrade
+
+> Update all your applications.
```

Comment:
We usually try to avoid 2nd person. How about "Update all applications in the system"?

Reply:
Ok, thanks for letting me know!